### DISCUSSION TITLE
helix and V

### DISCUSSION THREAD

**Discussion by cooperspencer** [vlang/v#14799] [General]
Hi,
does anyone use [helix](https://github.com/helix-editor/helix) to write V code? I try to do more stuff in helix but V, at least for me, doesn't that well with it. In the `languages.toml` from helix, V is included and I moved the language server to `~/.local/bin`, but the autocomplete doesn't work at all. Can anyone help me out?

**Comment by xbol0** [comment]
Have you fix it? I found my Helix can not render right syntax.
